### DISCUSSION TITLE
XNEdit 1.6.0

### DISCUSSION THREAD

**Discussion by unixwork** [unixwork/xnedit#148] [General]
version 1.6.0 (2024-12-05)
--------------------------

 - added IO filters
 - added gzip/bzip2 default options for opening and saving of compressed files
 - added a filter for monospace fonts to the font selector
 - added preferences for undo limits
 - added fallbackCharset preference
 - improved filedialog error handling
 - improved fillParagraph to be utf-8 aware
 - improved clipboard_to_string macro function to support utf-8
 - improved tolower/toupper macro functions to support utf-8
 - improved xnedit unicode-capable textfield and enabled it by default
 - improved text rendering and resizing performance
 - improved opening of large files
 - changed max tab size to 256
 - removed dependency on libpcre for matching editorconfig patterns
 - fixed encoding when including files
 - fixed crash in filename_dialog macro function
 – fixed compatibility with Motif 2.3.8
 - fixed detached tab window size
 - fixed font width detection
 - fixed filedialog not showing the default name when available
 - fixed some memory leaks and uninitialized memory access
 - fixed crash on Solaris sparc
 - fixed build on Solaris/Illumos
 - fixed integer overflow/possible buffer overflow in measureVisLine
 - fixed some UI color inconsistencies
 - fixed some color dialog rendering issues
 - fixed encoding error dropdown menu not working sometimes

<hr /><em>This discussion was created from the release <a href='https://github.com/unixwork/xnedit/releases/tag/v1.6.0'>XNEdit 1.6.0</a>.</em>

**Comment by mhk13** [comment]
Any chance to publish a compiled binaries for centos 7 & 9. My machines have no internet access from within and I can't install the required dependencies to build it.
Many thanks in advance

**Reply by unixwork** [reply]
I've added binaries for centos 7 for now, binaries for other systems will follow.

**Reply by mhk13** [reply]
many thanks!

**Reply by unixwork** [reply]
Added more binaries for EL8, EL9 and a distribution independent AppImage.

**Reply by mhk13** [reply]
> Added more binaries for EL8, EL9 and a distribution independent AppImage.

Thanks a lot, really appreciate it

**Comment by mhk13** [comment]
@unixwork 
Hello again, would really appreciate it you could add the binaries for centos 7 like before?
thank you in advance, apologies for the trouble

**Reply by unixwork** [reply]
Sorry, I had no time for that yet, I will create binaries this weekend.

**Reply by mhk13** [reply]
Many thanks bro, really appreciate it.

**Reply by unixwork** [reply]
Binaries are now available.
